modules/virt-vm-console-web.adoc

```diff
@@ -3,25 +3,22 @@
 // * virt/virtual_machines/virt-accessing-vm-consoles.adoc
 
 [id="virt-vm-console-web_{context}"]
```

Comment:
Can you also please update the id so that it's a closer match to title

Reply:
I mostly just care about adding the 'about', I leave the rest to your discretion :)